frontend/__tests__/unit/pages/About.test.tsx

```diff
@@ -35,6 +35,78 @@ jest.mock('utils/aboutData', () => ({
     'This is a test paragraph about the project.',
     'This is another paragraph about the project history.',
   ],
+  leaders: {
```

Comment:
These look like exact copies of what we have in `aboutData.ts`. Can we avoid the duplication?